autoscoper/src/ui/AutoscoperMainWindow.h

```diff
@@ -220,6 +222,10 @@ class AutoscoperMainWindow : public QMainWindow
   std::vector<unsigned int> textures;
   void reset_graph();
 
+  // Arrow key stepping
+  unsigned int stepDOF = 0;
```

Comment:
* `unsigned int` -> `int`
* rename from `stepDOF` to `currentDOF`
* move into `TimelineDockWidget`